### PR TITLE
Add camel-ignite wrapper

### PR DESCRIPTION
**Pull request by lopushen**
- no ignite feature, so added a dependency, had to wrap it as it is a regular jar
- left the comment in place as not sure that it is relevant for build phase

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -935,10 +935,11 @@
 
 <!--        Please refer to the component page (https://camel.apache.org/ignite.html) and to the Ignite docs (https://apacheignite.readme.io/docs/osgi-installation-in-karaf#preparatory-steps) for more information.]]>-->
 <!--        </details>-->
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Please either uncomment the details or remove them

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: uncommented, thanks

**Comment by essobedo** [comment]
There is a conflict to resolve

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**